adding persisted addition code

ArpitaBanerjeeDarnal opened the pull request.

LinusTyr reviewed (commented):
You have added tests (and they run successfully).

The numbers in the persisted-addition.html are not persisted though. Nor is persisted-addition.html using the logic that you are testing.

I need both of those things to be remedied.

ArpitaBanerjeeDarnal commented:
Made changes in the code according to suggestion

ArpitaBanerjeeDarnal commented:
Hi Linus,
Thank you so much for your help.
Actually the numberstore.js file is in dist,I compiled persisted-addition.html into numberstore.js using babel.The dist is in gitignore so it didn't get uploaded,but now changed it to src and updated successfully.
Hopefully now you can run the test also.

LinusTyr commented:
> Hi Linus, Thank you so much for your help. Actually the numberstore.js file is in dist,I compiled persisted-addition.html into numberstore.js using babel.The dist is in gitignore so it didn't get uploaded,but now changed it to src and updated successfully. Hopefully now you can run the test also.

I suspected as much. If you need to execute some CLI commands (beyond npm install, which is almost implied) to have the project end up in the "/dist" folder for example, you typically document these (and for ease of execution) in the package.json "scripts" segment. Please keep that in mind for the future.

LinusTyr reviewed (approved):
Good job!
